### ISSUE TITLE
security info page

### ISSUE THREAD

**Issue by jelletreep** [UtrechtUniversity/vre-docs#3]

**Comment by dometto** [comment]
Does #22 address this enough or should there be a separate page?

**Comment by jelletreep** [comment]
I don't remember what I had in mind when creating this issue :sweat_smile: 
We have now some tips about measures and best practices to keep a workspace secure. 

Maybe we need to clarify a bit more the security status of different workspace types, although this probably comes down to: SANE workspaces for highly sensitive data and 'the rest', or can we make some distinction in between? Is there a demand for that? I am thinking about e.g. datasets with commercial value. Maybe SANE is overkill, but would it be okay to use all other catalogue items or only certain catalog items...

So short answer, I think #22 addresses this enough, but maybe we can discuss at some point whether the "geschiktheidsclassficatie" the was done for the platform applies to all catalogue items, or if not, for which it applies, and which catalogue items are used at users responsibility?

**Comment by dometto** [comment]
> So short answer, I think https://github.com/UtrechtUniversity/vre-docs/pull/22 addresses this enough, but maybe we can discuss at some point whether the "geschiktheidsclassficatie" the was done for the platform applies to all catalogue items, or if not, for which it applies, and which catalogue items are used at users responsibility?

Yes, good idea. 👍 I've briefly mentioned SANE in #22, but we should think about how to present all the security aspects to users. Let's just keep this issue open as a reminder of that!
